AirCasting/MobilePeripheralSessionManager.swift

```diff
@@ -147,16 +145,18 @@ class MobilePeripheralSessionManager {
             do {
                 let existingStreamID = try storage.existingMeasurementStream(sessionUUID, name: stream.sensorName)
                 guard let id = existingStreamID else {
-                    return try self.createSessionStream(stream, sessionUUID)
+                    let streamId = try self.createSessionStream(stream, sessionUUID, storage: storage)
+                    try storage.addMeasurementValue(stream.measuredValue, at: location, toStreamWithID: streamId)
```

Comment:
no biggie, but since we're doing the `try storage.addMeasurementValue(stream.measuredValue, at: location, toStreamWithID: id)` anyway if guard is not hit, we can just change guard to `if` and only do the session stream creation inside (the measurement adding line will be executed in both cases then)